test/modules/Modules.Test/TestResultCoordinator/TestReportUtilTest.cs

```diff
@@ -101,12 +115,14 @@ public async Task TestGenerateTestResultReportsAsync_NullReportMetadata()
                     twinCountingReportMetadata,
                     deploymentReportMetadata,
                     directMethodReportMetadata,
-                    directMethodReportMetadataWithoutReceiverSource
+                    directMethodReportMetadataWithoutReceiverSource,
+                    edgeHubRestartMessageReportMetadata,
+                    edgeHubRestartDirectMethodReportMetadata
                 },
                 mockTestReportGeneratorFactory.Object,
                 mockLogger.Object);
 
-            Assert.Equal(expectedReportCount, reports.Length);
+            Assert.Equal(expectedExceptionCount, reports.Length);
```

Comment:
should not change this, right?

Reply:
I rename it since the `expectedReportCount` implies the number of report being generated, but that's not what the test case is counting. It is counting the total number of exceptions being thrown in this routine.

Reply:
TODO: I can rename it back.